nlu.py

```diff
@@ -20,6 +21,35 @@ def _get_commands(text_sequence: [str]) -> set:
     return possible_commands
 
 
+def _similarity(a: [str], b: [str]) -> float:
+    words = set(a + b)
+    vector_a = _get_vec(words, a)
+    vector_b = _get_vec(words, b)
+    return _consine_similarity(vector_a, vector_b)
+
+
+def _consine_similarity(a: [int], b: [int]) -> float:
+    if len(a) != len(b):
+        raise ValueError('list a and list b must have the same length')
```

Comment:
In case they are vector, not list
And error should contain critical information in case is the length

Reply:
done.